### PR TITLE
Add weight files for RNNoise

### PR DESCRIPTION
**Pull request by miaobin**
@huningxin @Honry PTAL, thanks!

### CODE REVIEW

File: models/rnnoise_nc/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+`.npy` files under this folder are tranned from project of [xiph/rnnoise](https://github.com/xiph/rnnoise), which is licensed under the BSD 3-Clause "New" or "Revised" License.
```
Comment (Honry) [inline models/rnnoise_nc/README.md]: Typo? `tranned`

Comment (Honry) [reply models/rnnoise_nc/README.md]: BTW, about the folder naming convention, shall we use `nchw` instead? As we don't have `nc` layout in spec although rnnoise is `nc` layout, @huningxin, WDYT?

**Comment by miaobin** [comment]
@Honry I have modified the folder names of two audio-related models.
PTAL, thanks!

**Review by Honry — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**